lexicon/conversion.py

```diff
@@ -81,10 +81,44 @@ def run(self):
 
 
 class LexiconUniqueOrthJob(Job):
-    def __init__(self, bliss_lexicon):
+    """Merge lemmata with the same orthography.
+
+    Merging strategy (only natural proper word lemmata are affected)
```

Comment:
This description should probably go into the constructor, where `merge_multi_orths_lemmata` is defined. Having it here is kind of confusing.